torch/nn/quantized/functional.py

```diff
@@ -1,4 +1,82 @@
+r""" Functional interface (quantized)."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import torch
 from torch._ops import ops
 
+def _extend_to_list(val, length=2):
+    if not isinstance(val, (tuple, list)):
+        val = [val] * length
+    return val
+
 relu = ops.quantized.relu
 add_relu = ops.quantized.add_relu
+
+
+def conv2d(input, weight, bias,
+           stride=1, padding=0, dilation=1, groups=1,
+           padding_mode='zeros',
+           scale=1.0, zero_point=0,
+           dtype=torch.quint8,
+           prepacked=True):
+    r"""
+    conv2d(input, weight, bias,
+           stride=1, padding=0, dilation=1, groups=1,
+           padding_mode='zeros',
+           scale=1.0, zero_point=0,
+           dtype=torch.quint8,
+           prepacked=True) -> Tensor
```

Comment:
Suggest having functionals only take in unpacked weights and do packing internally. Remove prepacked option for functional.